pkg/apiserver/registry/system/supportbundle/rest.go

```diff
@@ -213,7 +213,7 @@ func (r *supportBundleREST) collect(ctx context.Context, dumpers ...func(string)
 			return nil, err
 		}
 	}
-	outputFile, err := afero.TempFile(defaultFS, "", "bundle_*.tar.gz")
+	outputFile, err := afero.TempFile(defaultFS, "", "bundle_")
```

Comment:
Does OS append a random string as the suffix automatically? If not, what if we run multiple times of command "antctl supportbundle"? Does the latter file override the original one?

Reply:
Yes, it is. See
> // TempFile creates a new temporary file in the directory dir
// with a name beginning with prefix, opens the file for reading
// and writing, and returns the resulting *File.
// If dir is the empty string, TempFile uses the default directory
// for temporary files (see os.TempDir).
// Multiple programs calling TempFile simultaneously
// will not choose the same file.  The caller can use f.Name()
// to find the pathname of the file.  It is the caller's responsibility
// to remove the file when no longer needed.

Reply:
Got it, thanks for explaination.